### PR TITLE
Ensures that field is on screen behind the confirmation modal

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #10706 

##### ISSUE TYPE
 - Bugfix Pull Request
 - 
##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2021-07-22 at 12 32 09 PM

### CODE REVIEW

File: awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ describe('<NotificationTemplateListItem />', () => {
         <tbody>
           <NotificationTemplateListItem
             template={template}
+            onAddToach={jest.fn()}
```
Comment (nixocio) [inline awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js]: Hey @AlexSCorey was this change supposed to be part of this PR?
```suggestion
            onAddToast={jest.fn()}
```

File: awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js [hunk 2]
```diff
@@ -53,6 +54,7 @@ describe('<NotificationTemplateListItem />', () => {
...
```
Comment (nixocio) [inline awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js]: onAddToast
```suggestion
            onAddToast={jest.fn()}
```

File: awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js [hunk 3]
```diff
@@ -73,6 +75,7 @@ describe('<NotificationTemplateListItem />', () => {
...
```
Comment (nixocio) [inline awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js]:
```suggestion
            onAddToast={jest.fn()}
```

File: awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js [hunk 4]
```diff
@@ -94,6 +97,7 @@ describe('<NotificationTemplateListItem />', () => {
...
```
Comment (nixocio) [inline awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js]:
```suggestion
            onAddToast={jest.fn()}
```

File: awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js [hunk 5]
```diff
@@ -121,6 +125,7 @@ describe('<NotificationTemplateListItem />', () => {
                 },
               },
             }}
+            onAddToach={jest.fn()}
```
Comment (nixocio) [inline awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js]:
```suggestion
            onAddToast={jest.fn()}
```

**Review by nixocio — CHANGES_REQUESTED** [review]

**Review by nixocio — APPROVED** [review]

**Review by tiagodread — APPROVED** [review]

**Pull request merged**